benchmarks/large_exprs.py

```diff
@@ -74,7 +74,7 @@ def time_cse(self):
 
     def time_free_symbols(self):
 
-        self.super_long_expr.free_symbols
+        self.super_long_expr.free_symbols()
```

Comment:
`free_symbols` is an attribute not a method.

Reply:
Sorry, should've checked it better.